Makefile

```diff
@@ -59,6 +59,14 @@ verify-secure_api: secure_api.dir-verify
 verify: .verify-banner verify-ct verify-specs verify-code verify-secure_api
 	@echo $(CYAN)"\nDone ! Please check the verification output"$(NORMAL)
 
+verify-nss:
```

Comment:
I am not convinced, this should be part of our top-level Makefile, as it is NSS specific...
If I remember correctly the overall verification for the full HACL* (currently in NSS) is faster than the average NSS CI run, so it should not affect it. Am I correct ?

Reply:
We never verified everything in the NSS CI. The runtime of verifying Curve25519 is fine. But verifying everything will take quite a bit longer.

Reply:
Ok, I can take that as-is and move it later when I find a better place